<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivKeyPage</class>
 <widget class="QWidget" name="PrivKeyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>45</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
       <widget class="QValidatedLineEdit" name="payTo">
        <property name="toolTip">
         <string>The Phore address to send the payment to</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
		    <property name="placeholderText">
         <string>Enter a Phore address (e.g. PCYiHgGJJ6xGHqivmdZrYjRnhaYf6AJ2Mp)</string>
        </property>
       </widget>
      </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="copyAddress">
       <property name="toolTip">
        <string>Copy the currently selected address to the system clipboard</string>
       </property>
       <property name="text">
        <string>&amp;Copy</string>
       </property>
       <property name="icon">
        <iconset resource="../phore.qrc">
         <normaloff>:/icons/editcopy</normaloff>:/icons/editcopy</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="toolTip">
       </property>
       <property name="text">
        <string>C&amp;lose</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../phore.qrc"/>
 </resources>
 <connections/>
</ui>
